src/HotChocolate/Language/src/Language.SyntaxTree/Extensions/HashCodeExtensions.cs

```diff
@@ -46,4 +46,17 @@ public static void AddNodes(this ref HashCode hashCode, IReadOnlyList<ISyntaxNod
             hashCode.Add(nodes[i]);
         }
     }
+
+    public static void AddRange(this ref HashCode hashCode, params object?[] objects)
```

Comment:
This one is not good since we would create an array for each GetHashCode. I am removing it again.

Reply:
Ah yep that was a silly oversight good catch